etc/kayobe/ansible/templates/wazuh-secrets.yml.j2

```diff
@@ -7,7 +7,7 @@ secrets_wazuh:
   # Strengthen default wazuh api user pass
   wazuh_api_users:
     - username: "wazuh"
-      password: "{{ secrets_wazuh.wazuh_api_users[0].password | default(lookup('password', '/dev/null length=30' ), true) }}"
+      password: "{{ secrets_wazuh.wazuh_api_users[0].password | wazuh_password }}"
```

Comment:
I think it would be simpler to use:

https://docs.ansible.com/ansible/latest/collections/community/general/random_string_lookup.html

You can configure it to meet the complexity requirements.

Eg. `lookup('community.general.random_string', min_lower=1, min_upper=1, min_special=1, min_numeric=1)`

Or if installing that is a pain, you could use `lookup` to generate a few sub passwords, using limited characters sets, concatenate them together and then randomise the resultant string.

Reply:
I was thinking that having a small python script which will keep regenerating if requirements aren't met means that there is no chance of the password check failing and may even prevent the password being shown.

Reply:
It's better to rely on existing code if possible. I think random_string should be configurable to meet the requirements.